src/components/side_panel/pivot/pivot_layout_configurator/pivot_measure/pivot_measure.ts

```diff
@@ -81,4 +83,23 @@ export class PivotMeasureEditor extends Component<Props> {
       measure: this.props.measure,
     });
   }
+
+  getColoredSymbolToken(token: Token): Color | undefined {
+    if (token.type !== "SYMBOL") {
+      return undefined;
+    }
+    const tokenValue = token.value.startsWith("'")
+      ? token.value.slice(1, token.value.length - 1)
+      : token.value;
+    if (
+      this.props.definition.columns.some((col) => col.fieldName === tokenValue) ||
+      this.props.definition.rows.some((row) => row.fieldName === tokenValue) ||
```

Comment:
doesn't work with date(time) dimensions because it contains the granularity `'Created on:month_number'` :wink: 
```suggestion
      this.props.definition.columns.some((col) => col.nameWithGranularity === tokenValue) ||
      this.props.definition.rows.some((row) => row.nameWithGranularity === tokenValue) ||
```